identity-account-storage/src/types/encrypted_data.rs

```diff
@@ -0,0 +1,41 @@
+// Copyright 2020-2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use serde::Deserialize;
+use serde::Serialize;
+
+/// The structure returned after encrypting data
+#[derive(Debug, Clone, Deserialize, Serialize)]
+pub struct EncryptedData {
+  nonce: Vec<u8>,
+  associated_data: Vec<u8>,
+  tag: Vec<u8>,
+  ciphertext: Vec<u8>,
+}
+
+impl EncryptedData {
+  pub fn new(nonce: Vec<u8>, associated_data: Vec<u8>, tag: Vec<u8>, ciphertext: Vec<u8>) -> Self {
+    Self {
+      nonce,
+      associated_data,
+      tag,
+      ciphertext,
+    }
+  }
+
+  pub fn associated_data(&self) -> &[u8] {
+    &self.associated_data
+  }
+
+  pub fn ciphertext(&self) -> &[u8] {
+    &self.ciphertext
+  }
+
+  pub fn tag(&self) -> &[u8] {
+    &self.tag
+  }
+
+  pub fn nonce(&self) -> &[u8] {
```

Comment:
Would be nice with some more documentation here. Since just slices of bytes are returned it would be nice if the documentation explained (or links to something that explains) what `associated_data`, `tag`, `ciphertext` and `nonce` is/could be.